string  sensorName
float32 indicator
